### ISSUE TITLE
Using the Net after Training

### ISSUE THREAD

**Issue by jeya-maria-jose** [thomasneff/Generative-Adversarial-Network-based-Synthesis-for-Supervised-Medical-Image-Segmentation#3]
Hey , 

Can you please explan what does this sess.run() actually stores in this model and how exactly do you use the model for segmentation?

**Comment by thomasneff** [comment]
Hi,

> Can you please explan what does this sess.run() actually stores in this model 

I'm not sure what you mean by that?

In general: This code trains a DCGAN to simultaneously generate images and corresponding segmentation masks.

If you want to use the generated data as a supervised training input (i.e. use it as "groundtruth") for segmentation, you either have to take the trained Generator and use it as a data source for a segmentation network, or you have to sample some images from the generator and use those as an input for the segmentation network.

The GAN itself does not directly perform segmentation as its main objective, it is mainly used to generate additional training data (similar to data augmentation).
